SPECS/openvswitch.spec

```diff
@@ -40,12 +40,13 @@ Patch1000: openvswitch-2.17.7-comment-failing-tests.XCP-ng.patch
 Patch1001: openvswitch-2.17.7-add-pythonpath-ipsec.XCP-ng.patch
 Patch1002: openvswitch-2.17.7-CVE-2023-3966-netdev-offload-tc-Check-geneve-metadata-length.backport.patch
 Patch1003: openvswitch-2.17.7-CVE-2023-5366-odp-ND-Follow-Open-Flow-spec-converting-from-OF-to-DP.backport.patch
+Patch1004: ovs-ctl-set-default-ssl-ciphers-and-protocols.patch
 
 Requires(post): systemd
 Requires(preun): systemd
 Requires(postun): systemd
 BuildRequires: systemd
-BuildRequires: openssl, openssl-devel, python3
+BuildRequires: xs-openssl, xs-openssl-devel, xs-openssl-libs, python3
```

Comment:
Doesn't `xs-openssl-devel` already pull `xs-openssl-libs`?

Reply:
Indeed, I removed it.